lucifer/Shell/__init__.py

```diff
@@ -0,0 +1,61 @@
+import lucifer.Indexing as Indexing
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/manual/alexspam9090/Lucifer/pullRequest?prid=6170630)